test/e2e/dryrun.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+	"sigs.k8s.io/kwok/pkg/utils/path"
+)
+
+func loadExpectedClusterDetails(filepath string, clusterName string, rootDir string) (string, error) {
+	data, err := os.ReadFile(filepath)
+	if err != nil {
+		return "", err
+	}
+	out := string(data)
+	out = strings.ReplaceAll(out, "<CLUSTER_NAME>", clusterName)
+	out = strings.ReplaceAll(out, "<ROOT_DIR>", rootDir)
```

Comment:
We should replace kwokctl's output, not testdata

Reply:
you mean that I should replace the clusterName with <CLUSTER_NAME> in the output right?

Reply:
yes